manifests/piped/templates/deployment.yaml

```diff
@@ -19,9 +19,7 @@ spec:
         sidecar.istio.io/inject: "false"
         rollme: {{ randAlphaNum 5 | quote }}
     spec:
-      {{- if .Values.serviceAccount.create }}
```

Comment:
refs: https://github.com/pipe-cd/pipecd/pull/3535#discussion_r851001593

to use existing service account.